sdk/python/kfp/compiler/_default_transformers.py

```diff
@@ -12,16 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import uuid
+from typing import Dict, Optional, Text
 from ..dsl._container_op import BaseOp, ContainerOp
 
+# Pod label indicating the SDK type from which the pipeline is
+# generated. By default it's set to kfp.
+_SDK_ENV_LABEL = 'pipelines.kubeflow.org/pipeline-sdk-type'
+_SDK_ENV_DEFAULT = 'kfp'
+
+# Pod label representing the random pipeline ID. If two pods have the same
+# pipeline ID then they belong to the same KFP pipeline.
+_PIPELINE_UUID_LABEL = 'pipelines.kubeflow.org/pipeline-uuid'
+
+def get_default_telemetry_labels() -> Dict[Text, Text]:
+    """Returns the default pod labels for telemetry purpose."""
+    result = {
+        _SDK_ENV_LABEL: _SDK_ENV_DEFAULT,
```

Comment:
What if we also include the SDK version (`kfp.__version__`)?

Reply:
That's also doable. Let me update.

Reply:
> This will break the cache though....

This might be serious. I'll try to think of a solution.

Reply:
I have excluded the uuid in this PR, so I guess it's safe for caching per se.